### ISSUE TITLE
aerospike_ycsb run failed

### ISSUE THREAD

**Issue by ehankland** [GoogleCloudPlatform/PerfKitBenchmarker#697]
Got this error message:
```
RemoteCommandError: Got non-zero return code (1) executing scp -P 22 -pr -2 -o UserKnownHostsFile=/dev/null -o StrictHostKeyChecking=no -o IdentitiesOnly=yes -o PreferredAuthentications=publickey -o PasswordAuthentication=no -o ConnectTimeout=5 -o GSSAPIAuthentication=no -o ServerAliveInterval=30 -o ServerAliveCountMax=10 -i /tmp/perfkitbenchmarker/run_b4c9c699/perfkitbenchmarker_keyfile (home_dir)/PerfKitBenchmarker/perfkitbenchmarker/data/ycsb/workloada perfkit@23.236.56.191:/tmp/pkb/workloada
STDOUT: STDERR: Warning: Permanently added '23.236.56.191' (ECDSA) to the list of known hosts.^M
Connection closed by 23.236.56.191^M
lost connection
```
Looks like a problem copying files to the VM - a simple retry may fix this. It's also possible that the VM was already stopped because we set `--maintenance-policy TERMINATE` when we create them. A similar run passed, so this is just a flakiness issue.

**Comment by ehankland** [comment]
Same thing happened with cassandra_ycsb

**Comment by voellm** [comment]
@cmccoy  - Can you see if there is a quick fix?

**Comment by cmccoy** [comment]
Sure. The instances weren't terminated. I'll add a retry.

**Comment by cmccoy** [comment]
Opened #699. Testing now.

**Comment by cmccoy** [comment]
Closed by #699.
